src/backend/distributed/sql/downgrades/citus--11.0-1--10.2-4.sql

```diff
@@ -40,7 +40,7 @@ COMMENT ON FUNCTION master_append_table_to_shard(bigint, text, text, integer)
 GRANT ALL ON FUNCTION start_metadata_sync_to_node(text, integer) TO PUBLIC;
 GRANT ALL ON FUNCTION stop_metadata_sync_to_node(text, integer,bool) TO PUBLIC;
 
-DROP FUNCTION pg_catalog.citus_disable_node(nodename text, nodeport integer, force bool);
```

Comment:
we should use the 11.0-1--11.0-2 upgrade/downgrade path.

Reply:
I did that, but to be honest I couldn't follow why we ended up 11.0-2 just yet.

In general, what is the benefit of having 11.0-2 at this point (e.g., before the release)?

Reply:
upgrade path from beta